sewer/cli.py

```diff
@@ -125,6 +125,22 @@ def main():
         except KeyError as e:
             logger.info("ERROR:: Please supply {0} as an environment variable.".
                         format(str(e)))
+
+    if dns_provider == 'aurora':
+        from . import AuroraDns
+        try:
+            AURORA_API_KEY = os.environ['AURORA_API_KEY']
+            AURORA_SECRET_KEY = os.environ['AURORA_API_KEY']
```

Comment:
@wilfriedjonker Hi, 
should this line be:
`AURORA_SECRET_KEY = os.environ['AURORA_SECRET_KEY']` ? 
Or does auroraDNS use the same value for Api-key and Secret-key?